core/scripts/chaincli/handler/keeper.go

```diff
@@ -414,8 +414,12 @@ func (k *Keeper) deployUpkeeps(ctx context.Context, registryAddr common.Address,
 		if err != nil {
 			log.Fatal(upkeepId, upkeepAddr.Hex(), ": AddFunds failed - ", err)
 		}
-		k.waitTx(ctx, addFundsTx)
-		log.Println(upkeepId, upkeepAddr.Hex(), ": Upkeep funded - ", helpers.ExplorerLink(k.cfg.ChainID, addFundsTx.Hash()))
+		addFundsTxStatus := k.waitTx(ctx, addFundsTx)
```

Comment:
If you changed this function to return an error, you could then rework this to not use an `else` block, e.g.:

```golang
if err := k.waitTx(ctx, addFundsTx); err != nil {
    log.Fatal("AddFunds failed for upkeepId: ", upkeepId, ", and upkeepAddr: ", upkeepAddr.Hex())
}

log.Println(upkeepId, upkeepAddr.Hex(), ": Upkeep funded - ", helpers.ExplorerLink(k.cfg.ChainID, addFundsTx.Hash()))
```